src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -31,7 +32,9 @@ object DplaMapData {
 case class DplaMapData(
                          sourceResource: DplaSourceResource,
                          edmWebResource: EdmWebResource,
-                         oreAggregation: OreAggregation
+                         oreAggregation: OreAggregation,
+                         // Add id property at the root level (following ES schema)
+                         id: ExactlyOne[String]
```

Comment:
Hmm, I'm basing it off of the DPLA MAP spec. I agree if there needs to be a standalone ID, we need a spot for it.